### PR TITLE
[VM] Remove gating by AptosVersion

### PR DESCRIPTION
**Pull request by kris524**
## Motivation

These are the cleanup from old Diem versions

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan

Haven't done tests, will wait to see if this is the right track of changes I need to make first

## Related PRs

None

### CODE REVIEW

**Comment by kris524** [comment]
Apologies, it seems I have my other two commits from a different task in here....I need to figure out how to fix that

File: aptos-move/aptos-vm/src/aptos_vm.rs [hunk 1]
```diff
@@ -244,11 +240,10 @@ impl AptosVM {
                     }
                 }
                 TransactionPayload::ScriptFunction(script_fn) => {
-                    let aptos_version = self.0.get_version()?;
+                    // let aptos_version = self.0.get_version()?;
```
Comment (davidiw) [inline aptos-move/aptos-vm/src/aptos_vm.rs:243]: delete

**Review by davidiw — COMMENTED** [review]
overall looks good, you can use 

`git rebase -i HEAD~4` and delete the commits
please make sure to sign our cla: https://github.com/aptos-labs/aptos-core/blob/main/documentation/contributing/individual-cla.pdf
e-signatures are fine

**Comment by kris524** [comment]
I can't figure out git...

**Comment by davidiw** [comment]
@kris524 , hit me up on discord @davidiw

**Comment by davidiw** [comment]
Merged in another PR.

**Pull request closed without merging**